components/ButtonBar/tests/unit/ButtonBarObservationTests.swift

```diff
@@ -0,0 +1,167 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import XCTest
+import MaterialComponents.MaterialButtonBar
+import MaterialComponents.MaterialButtons
+
+class ButtonBarObservationTests: XCTestCase {
+
+  var buttonBar: MDCButtonBar!
+
+  override func setUp() {
+    buttonBar = MDCButtonBar()
+  }
+
+  func testAccessibilityLabelChangesAreObserved() {
+    let item = UIBarButtonItem(title: "Title", style: .plain, target: nil, action: nil)
+    item.accessibilityLabel = "Label"
+    buttonBar.items = [item]
+    buttonBar.layoutSubviews()
+
+    do {
+      let accessibilityLabels =
+          buttonBar.subviews.flatMap { $0 as? MDCButton }.flatMap { $0.accessibilityLabel }
+      XCTAssertEqual(accessibilityLabels, [item.accessibilityLabel!])
+    }
+
+    // Change the value post-assignment
+    item.accessibilityLabel = "Other label"
```

Comment:
Please make these separate tests (unless it's already covered elsewhere):
1. Initial values from UIButtonBarItem match MDCButton 
2. KVO changes are propagated

Reply:
Done.